File: app/src/main/java/com/arctouch/codechallenge/ui/adapter/presenter/HomeAdapterPresenter.java
package com.arctouch.codechallenge.ui.adapter.presenter;

import com.arctouch.codechallenge.entity.Movie;

import java.util.List;

public interface HomeAdapterPresenter {

    //

    interface View {
        void refreshMoviesList(List<Movie> movies);
        void addMoviesToList(List<Movie> movies);
    }
}

File: app/src/main/java/com/arctouch/codechallenge/ui/adapter/presenter/HomeAdapterPresenter.kt
package com.arctouch.codechallenge.ui.adapter.presenter

import com.arctouch.codechallenge.entity.Movie

interface HomeAdapterPresenter {

    //

    interface View {
        fun refreshMoviesList(movies: List<Movie>)
        fun addMoviesToList(movies: List<Movie>)
    }
}
